Add some website content blocks

dmlb2000 opened the pull request:
This adds some content blocks to support the main website under construction.

Signed-off-by: David Brown <dmlb2000@gmail.com>

dmlb2000 committed:
Add some website content blocks

Signed-off-by: David Brown <dmlb2000@gmail.com>

dmlb2000 committed:
Merge branch 'master' into add-project-docs

dmlb2000 commented:
@kariesue I've not seen any review on this yet, did you have time to look at this?

kariesue commented on project/mission-vision.md:
```diff
@@ -0,0 +1,14 @@
+## Mission
+
+To provide institutions a foundation to support the scientific data
+life-cycle.
```
lifecycle

BenjaminStuermer replied:
According to dictionaries I've checked on line, "life cycle" should be written as two words without a hyphen. (https://www.dictionary.com/browse/life-cycle)

dmlb2000 replied:
Done.

kariesue commented on project/mission-vision.md:
```diff
@@ -0,0 +1,14 @@
+## Vision
+
+We will become a focal point for institutions to collaborate, share
+scientific data, integrate new technologies and develop their
```
comma after technologies

dmlb2000 replied:
So are you advocating for an Oxford Comma? https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

dmlb2000 replied:
But Weird Al told me not to (https://www.youtube.com/watch?v=8Gv0H-vPoDc)

kariesue replied:
Oh yes! I loooooove the Oxford comma! :)

dmlb2000 replied:
Okay, good enough for me ;)

kariesue commented on project/mission-vision.md:
```diff
@@ -0,0 +1,14 @@
+We will become a focal point for institutions to collaborate, share
+scientific data, integrate new technologies and develop their
+research portfolio. We will enable institutions to meet and exceed
+emerging sponsor data challenges, while maintaining the promises
```
remove "the" before promises

dmlb2000 replied:
Done.

kariesue commented on project/mission-vision.md:
```diff
@@ -0,0 +1,14 @@
+research portfolio. We will enable institutions to meet and exceed
+emerging sponsor data challenges, while maintaining the promises
+kept in the past. We will provide the platform to accellerate
+scientific research in future directions never thought possible
```
We will provide the platform to accelerate scientific research in directions never before thought possible.

dmlb2000 replied:
Done.

kariesue commented on project/website.md:
```diff
@@ -0,0 +1,35 @@
+require community to share ideas, processes and support features to
+solve the requirements. This is the natural evolution of Open Source
+to Open Everything.
+
```
Open source does not end with the release of the source code. Complex problems require the community to share ideas, processes, and support features to solve the requirements -- the evolution from open source, to open everything.

kariesue replied:
Needs a line about how Pacifica enables this community interaction ...

dmlb2000 replied:
This should be done and ready for second review.

kariesue commented on project/website.md:
```diff
@@ -0,0 +1,35 @@
+Pacifica has solutions to manage data between projects that are
+completing and collaborative. Pacifica can accelerate the scientific
+process by making search and discovery tracta
+
```
Scientific data management systems for research and development institutions need to provide uncomplicated solutions to meet often complicated problems. Pacifica offers institutions a platform to collaboratively manage data between projects. 

... looks like unfinished section. LMK when to take another look at it.

dmlb2000 replied:
Yup I screwed up on this... didn't complete the section good catch.

dmlb2000 replied:
Done.

kariesue commented on project/website.md:
```diff
@@ -0,0 +1,35 @@
+### Impact
+
+Pacifica seeks to enable institutions to reduce risk while
+accelerating scientific impact of the data they produce. Pacifica has
```
is it scientific impact for everyone who might use Pacifica?

dmlb2000 replied:
Done.

kariesue commented on project/website.md:
```diff
@@ -0,0 +1,35 @@
+
+Pacifica seeks to enable institutions to reduce risk while
+accelerating scientific impact of the data they produce. Pacifica has
+interfaces to allow scientists to automate workflow to ensure
```
legal doesn't want us using the word "ensure." Consider safeguard, provide, ...

dmlb2000 replied:
Done.

dmlb2000 committed:
Merge branch 'master' into add-project-docs

dmlb2000 committed:
Address Karen's Suggestions

This should hopefully address all of Karen's suggestions about the
text for the website.

Signed-off-by: David Brown <dmlb2000@gmail.com>

dmlb2000 commented:
@kariesue If you could review again, I should have fixed everything requested.
